src/FeeSchedule.js

```diff
@@ -0,0 +1,62 @@
+import * as proto from "@hashgraph/proto";
+export default class FeeSchedule {
+    /**
+    * @param {object} props
+    * @param {any} props.transactionFeeSchedule
```

Comment:
You shouldn't be using `any` or `object` for any fields that are not `props`. This should be field should be of type `TransactionFeeSchedule` and `expiryTime` should be renamed to `expirationTime` and be of type `Timestamp`